Sim/SimDelphesInterface/src/DelphesSimulation.h

```diff
@@ -3,10 +3,14 @@
 
 #include "TObjArray.h"
 #include "GaudiAlg/GaudiAlgorithm.h"
+#include "datamodel/MCParticleCollection.h"
+#include "datamodel/GenVertexCollection.h"
+//#include "datamodel/GenJetCollection.h"
 #include "datamodel/ParticleCollection.h"
+#include "datamodel/ParticleMCParticleAssociationCollection.h"
 #include "datamodel/GenJetCollection.h"
+#include "datamodel/GenJetParticleAssociationCollection.h"
 #include "datamodel/METCollection.h"
-#include "datamodel/MCParticleCollection.h"
 #include "TFile.h"
 #include "FWCore/DataHandle.h"
 #include "HepMC/GenEvent.h"
```

Comment:
I think several of these could just be forward declarations and then only included in the implementation file.